### DISCUSSION TITLE
Chrome Extension for Chitchatter

### DISCUSSION THREAD

**Discussion by MohammadTaseenKhan** [jeremyckahn/chitchatter#41] [Show and tell]
@jeremyckahn 

I am currently working on chrome extension where you can quickly navigate to it by only one click. Where you don't need to type URL manually.

It's window will be responsive and will not have any overflow on it you will be able to do the same which are used to do in the browser tab.

**Comment by jeremyckahn** [comment]
That sounds really cool! Please share your progress here as you develop the extension. 🙂

**Reply by MohammadTaseenKhan** [reply]
[https://github.com/MohammadTaseenKhan/Chrome-Extention-For-Chitchatter.git](https://github.com/MohammadTaseenKhan/Chrome-Extention-For-Chitchatter.git)

**Comment by MohammadTaseenKhan** [comment]
@jeremyckahn 

Preview

![image](https://user-images.githubusercontent.com/85288261/195616396-b93722d6-8727-4bdb-a191-5a40f4505e73.png)
![image](https://user-images.githubusercontent.com/85288261/195616764-5dfb862c-f24c-49b5-937d-fdce3e791e14.png)

**Reply by jeremyckahn** [reply]
Whoa this is awesome @MohammadTaseenKhan! Thanks for the work you've done here. I am quite busy over the next few days, but I will take a closer look at your repo as soon as I can. Let me know if you need any support from me for the extension!
